ddl/ddl_worker.go

```diff
@@ -462,6 +461,16 @@ func (w *worker) handleDDLJobQueue(d *ddlCtx) error {
 	}
 }
 
+func writeBinlog(binlogCli *pumpcli.PumpsClient, txn kv.Transaction, job *model.Job) {
+	if job.IsDone() || job.IsRollbackDone() ||
+		// Drainer receives a binlog without this column of data when the job is in the "Delete only" state.
```

Comment:
```suggestion
		// Drainer receives a binlog without this column of data when the job is in the "DeleteOnly" state.
```
How about "DeleteOnly" or "delete only"? BTW the comment sounds awkward, can you refine it?

Reply:
Any suggestions for this comment?